app/views/comfy/admin/cms/files/_file.html.haml

```diff
@@ -1,4 +1,7 @@
 %li{data: {id: file.id}}
+  :ruby
+    file_tag = "{{ cms:file_link #{file.id} }}"
```

Comment:
I think I want the same dealio to generate `as: image` for image attachments. Will expand that helper (or make new one) to do this.